test/web-platform-tests/to-run.yaml

```diff
@@ -1191,14 +1190,14 @@ template-element/template-content-hierarcy.html: [fail, "We do not implement htt
 
 DIR: html/semantics/selectors
 
-pseudo-classes/dir-html-input-dynamic-text.html: [fail, need nwsapi fix]
-pseudo-classes/dir.html: [fail, https://github.com/dperini/nwsapi/issues/45]
+pseudo-classes/dir-html-input-dynamic-text.html: [fail, Unknown]
+pseudo-classes/dir.html: [fail, Unknown]
 pseudo-classes/focus-autofocus.html: [timeout, Unknown]
 pseudo-classes/focus.html: [timeout, Unknown]
-pseudo-classes/indeterminate-type-change.html: [fail, Unknown]
 pseudo-classes/readwrite-readonly-type-change.html: [fail, Unknown]
 pseudo-classes/readwrite-readonly.html: [fail, Unknown]
-pseudo-classes/required-optional-hidden.html: [fail, Unknown]
+pseudo-classes/required-optional-hidden.html: [fail, Incorrect test]
```

Comment:
There seems to be a bug in the testcase.
<input type="hidden"> should not match `:optional` pseudo-class.
2nd testcase should result `rgb(0, 255, 0)`.